index.html

```diff
@@ -184,6 +184,21 @@ <h2> About <br> Myself </h2>
       </div>
     </section>
   </main>
+  <footer class="">
+    <h2 id="contact">Contact me</h2>
+    <p>
+      If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+      <br>
+      I’d love to help with it!
+    </p>
+    <form id="contact-form" action="https://formspree.io/f/xgedkalg" method="POST">
+      <input type="text" name="name" placeholder="Lorem Ipsum" maxlength="30" required>
+      <input type="email" name="email" id="email" placeholder="loremipsum@gmail.com" required>
+      <textarea name="message" id="message" placeholder="Write your message here..." cols="30" rows="10" maxlength="500" required></textarea>
+      <button class="btn" form="contact-form" type="submit">Get in touch</button>
```

Comment:
- [ ] 👀 Your button has a visually appealing appearance, but there are some adjustments needed to match the design guidelines more accurately. I recommend reviewing the design carefully and updating your code accordingly to ensure it aligns with the specified design. Pay attention to details like colors, font styles, and any additional design elements that may be required.
![image](https://github.com/razvanso/rio/assets/55840999/abb221f5-4b7f-490d-ba78-23059e80fc2d)

Please make the necessary changes to achieve the desired design outcome. Keep up the good work! 👍🎨

Reply:
@shahadat3669 , the color of the border was indeed wrong - **you have good eyes** :eyes: !
In the process, I also added the color variables from Figma (which we'll probably need later) and removed some extra code, including the additional outline for the hover/focus states, which I found useful for the user experience, but were not part of the initial design.